inference-engine/tests/unit/vpu/base/graph_transformer_tests.hpp

```diff
@@ -88,6 +89,9 @@ class TestModel final {
     void createInputs(std::vector<DataDesc> inputDescs);
     void createOutputs(std::vector<DataDesc> outputDescs);
 
+    void createInputs(int numInputs);
+    void createOutputs(int numOutputs);
```

Comment:
IMO, I see no good reason to create these methods in common `graph_transformer_tests.hpp`
You use `createInputs` only with `1` value everywhere and `createOutputs` with `1` or `2`.
Using only `_dataDesc` may not be a generalization for all cases of `TestModel` usage, I guess. 

As a result, I'd prefer to implement such methods in adjust_data_batch scope

Reply:
I see your point. I will implement the methods as you suggest.